app/frontend/src/components/Alert.tsx

```diff
@@ -26,10 +26,9 @@ export default function Alert({
 }: AlertProps) {
   const classes = useStyles();
 
-  const oldErrorKey = Object.keys(
-    grpcErrorStrings
-  ).find((oldError): oldError is ObscureGrpcErrorMessages =>
-    children.includes(oldError)
+  const oldErrorKey = Object.keys(grpcErrorStrings).find(
+    (oldError): oldError is ObscureGrpcErrorMessages =>
+      children?.toString().includes(oldError) ?? false
```

Comment:
We aren't really meant to interact with `children` directly this way if it's openly allowing any `ReactNode`. It'd be safer to use one of the [`React.Children` utility](https://reactjs.org/docs/react-api.html#reactchildren) on `children` first and apply this to the individual child element.

@lucaslcode I think we tried to keep it open to accept `ReactNode` but had to change this to only allow string, otherwise it doesn't map the obscure gprc error correctly to the nicer ones right?

Reply:
OK, I've decided not to touch it, not worth it